.github/workflows/codespell.yml

```diff
@@ -0,0 +1,16 @@
+# GitHub Action to automate the identification of common misspellings in text files.
+# https://github.com/codespell-project/actions-codespell
+# https://github.com/codespell-project/codespell
+name: codespell
+on: [push, pull_request]
+jobs:
+  codespell:
+    name: Check for spelling errors
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: codespell-project/actions-codespell@master
+        with:
```

Comment:
Can we add the [`only_warn=1`](https://github.com/codespell-project/actions-codespell#parameter-only_warn) parameter? Spelling usually isn't vital and I don't want contributors to feel discouraged over it.

Reply:
Done. Fair enough. we could for example split this up, so typos in one part didn't matter, but e.g. those in the presets did, or would you rather just keep it all flexible?

Reply:
All flexible sounds good. It's nice to get the warnings, but I don't think the build should fail over spelling.

Reply:
I've just realised I misspoke. Even with only_warn=0, it's still down to your GitHub config if the build fails or not and if can be merged. For example this:
https://github.com/codespell-project/codespell/pull/1672

So you could see the spelling failure and get the annotations, but still merge it. Or are you worried the red cross will put people off or confuse them?